draft-ietf-ppm-dap.md

```diff
@@ -2751,7 +2737,7 @@ structure includes the following:
 
 Once the `Leader` has constructed a `CollectionJobResp` for the Collector, the
 Leader considers the batch to be collected, and further aggregation jobs MAY NOT
```

Comment:
I'm pretty sure MAY NOT should be MUST NOT

Reply:
Good catch!